src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,89 @@
+@use "../../variables" as *;
+.contact {
+  text-align: center;
+  padding: 32px;
+  @media screen and (min-width: $breakpoint-md) {
+    padding: 52px;
+  }
+  .logo-wrapper {
+    display: block;
+    margin-bottom: 20px;
+    & > img {
+      width: 150px;
+    }
+  }
+  .footer-more {
+    padding-left: 7px;
+    padding-right: 7px;
+  }
+  .footer-more {
+    align-self: flex-end;
+    cursor: pointer;
+    color: $black;
+    text-decoration: none;
+    font-size: $fontsize-s;
+    transition: color 0.2s ease;
+    @media screen and (min-width: $breakpoint-md) {
+      &:hover {
+        color: $gray_nobel;
+      }
+    }
+  }
+}
+.text-wrap {
+  font-size: $fontsize-s;
+  margin-bottom: 20px;
+}
+.molecules-footer {
+  padding: 16px;
+  @media screen and (max-width: $breakpoint-md) {
+    gap: 8px;
+  }
+  background-color: $white_smoke;
+  font-size: $fontsize-s;
+  border-top: 1px solid $gray_nobel;
+  .footer-column {
+    display: flex;
+    justify-content: center;
+    padding: 0px;
+    margin-bottom: 0px;
+    @media screen and (max-width: $breakpoint-md) {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      gap: 8px;
+    }
+    & > li {
+      list-style: none;
+    }
+    .footer-each-content {
+      display: block;
+      text-decoration: none;
+      color: $black;
+      transition: color 0.2s ease;
+      @media screen and (min-width: $breakpoint-md) {
+        &:hover {
+          color: $orange_bistre;
+        }
+        &.disabled-line {
+          &::after {
+            content: "";
+          }
+        }
```

好みの領域かもですがこちらの方がシンプルにかけると思います。
```suggestion
        &.disabled-line::after {
            content: "";
        }
```

---

なるほど！そちらにします。